src/assets/iconAe.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Check [this](https://github.com/aeternity/superhero-ui/pull/897#discussion_r556475843) discussion.